monster-cards/src/app.ts

```diff
@@ -0,0 +1,55 @@
+import { DNode, WidgetOptions, WidgetState } from 'dojo-widgets/interfaces';
+import d from 'dojo-widgets/d';
+import createProjector, { Projector } from 'dojo-widgets/createProjector';
+
+import createNavbar from './widgets/navbar/createNavbar';
+import createHomePage from './widgets/home/createHomePage';
+import createCardDetailsPage from './widgets/card-details/createCardDetailsPage';
+import createCardsPage from './widgets/cards/createCardsPage';
+import createGameplayPage from './widgets/gameplay/createGameplayPage';
+import createAboutPage from './widgets/about/createAboutPage';
+
+function getPageRoute(instance: any) {
```

Comment:
Perhaps rename to `getPageFromRoute` and change the `let route` to `let page` as you're returning a page, not a route.